src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,145 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

It will be initialized by 0 by default